api/src/main/java/com/velocitypowered/api/util/ProtocolConstants.java

```diff
@@ -0,0 +1,100 @@
+package com.velocitypowered.api.util;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ProtocolConstants {
```

Comment:
This can be replaced with `ServerPing.Version` constants? (maybe move that class to a separate package) Now you are going to store the friendly version name twice and feels like duplicate code.

Reply:
In any case, I don't like the enum name `ProtocolConstants`, something like `ProtocolVersion` would be more fitting.

Reply:
100% agree with the name change. As per replacing it with `ServerPing.Version`, if anything `ProtocolConstants` should replace it since the functionality is essentially the same, but with more function